releasenotes/notes/add-primitives-support-31af39549b5e66e3.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    The :class:`~.GroverOptimizer` class has a new keyword argument, ``sampler`` which is
+    used to run the algorithm using an instance of the :class:`~.BaseSampler`
+    interface to calculate the results. This new argument supersedes
+    the ``quantum_instance`` argument and accordingly, ``quantum_instance``
+    is pending deprecation and will be deprecated and subsequently removed in
+    future releases.
+  - |
+    The :class:`~.MinimumEigenOptimizer` class takes the primitives-based algorithms
+    (``qiskit.algorithms.minimum_eigensolvers.SamplingVQE`` and
```

Comment:
In theory it supports any subclass of `SamplingMinimumEigensolver` right (that says it supports aux_ops ie has a valid eigenstate value in the result. `SamplingVQE` is all there is for now but it could be cited as an example of the former.

Reply:
You are right. It's my mistake. I will fit it.